tests/Nox.ClientApi.Tests/Generated/Nox.Generator/Nox.Generator.NoxCodeGenerator/Application.Commands/UpsertWorkplaceEnumerationsTranslationsCommand.g.cs

```diff
@@ -19,82 +19,74 @@
 using WorkplaceEntity = ClientApi.Domain.Workplace;
 
 namespace ClientApi.Application.Commands;
-public partial record  UpsertWorkplacesOwnershipsTranslationsCommand(IEnumerable<WorkplaceOwnershipLocalizedDto> WorkplaceOwnershipLocalizedDtos) : IRequest<IEnumerable<WorkplaceOwnershipLocalizedDto>>;
+public partial record UpsertWorkplacesOwnershipsTranslationCommand(Enumeration Id, WorkplaceOwnershipLocalizedUpsertDto WorkplaceOwnershipLocalizedUpsertDto, CultureCode CultureCode) : IRequest<WorkplaceOwnershipLocalizedKeyDto>;
 
-internal partial class UpsertWorkplacesOwnershipsTranslationsCommandHandler : UpsertWorkplacesOwnershipsTranslationsCommandHandlerBase
+internal partial class UpsertWorkplacesOwnershipsTranslationCommandHandler : UpsertWorkplacesOwnershipsTranslationCommandHandlerBase
 {
-	public UpsertWorkplacesOwnershipsTranslationsCommandHandler(
+	public UpsertWorkplacesOwnershipsTranslationCommandHandler(
         IRepository repository,
 		NoxSolution noxSolution) : base(repository, noxSolution)
 	{
 	}
 }
-internal abstract class UpsertWorkplacesOwnershipsTranslationsCommandHandlerBase : CommandCollectionBase<UpsertWorkplacesOwnershipsTranslationsCommand, WorkplaceOwnershipLocalized>, IRequestHandler<UpsertWorkplacesOwnershipsTranslationsCommand, IEnumerable<WorkplaceOwnershipLocalizedDto>>
+internal abstract class UpsertWorkplacesOwnershipsTranslationCommandHandlerBase : CommandBase<UpsertWorkplacesOwnershipsTranslationCommand, WorkplaceOwnershipLocalized>, IRequestHandler<UpsertWorkplacesOwnershipsTranslationCommand, WorkplaceOwnershipLocalizedKeyDto>
 {
+	
 	public IRepository Repository { get; }
-	public UpsertWorkplacesOwnershipsTranslationsCommandHandlerBase(
+	public UpsertWorkplacesOwnershipsTranslationCommandHandlerBase(
         IRepository repository,
 		NoxSolution noxSolution) : base(noxSolution)
 	{
 		Repository = repository;
 	}
 
-	public virtual async Task<IEnumerable<WorkplaceOwnershipLocalizedDto>> Handle(UpsertWorkplacesOwnershipsTranslationsCommand command, CancellationToken cancellationToken)
+	public virtual async Task<WorkplaceOwnershipLocalizedKeyDto> Handle(UpsertWorkplacesOwnershipsTranslationCommand command, CancellationToken cancellationToken)
 	{
+		System.Diagnostics.Debug.WriteLine("UpsertTranslationCommandHandle");
```

Comment:
Was this added by mistake?

Reply:
It was for debug purposes during the implementation, removed.